src/xcode/ENA/ENA/Source/Extensions/__tests__/String+TodayTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+@testable import ENA
+import Foundation
+import XCTest
+
+final class StringTodayTests: XCTestCase {
+	func testTodayFormat() {
+		let formatted = String.formattedToday()
+		XCTAssertNotNil(formatted.range(of: #"\d{4}-\d{2}-\d{2}"#, options: .regularExpression), "Expected format to be in yyyy-MM-dd")
```

Comment:
We had a terrible bug in the beginning. The format `formattedToday` used internally was:

```
"yyyy-mm-dd"
```

This gave is dates like:

```
2020-01-23
2020-02-23
2020-13-23
```

but also:

```
2020-56-23
// and
2020-59-23
```
Your regex would not catch that bug. Right? :) 

The correct format would have been:

```
"yyyy-MM-dd"
```

(uppercase m)

We even went so far at some point to write our custom date formatter... Sadly `formattedToday` is not really testable because it uses `Date()`... Not sure if it is worth it changing that now though. Approved.

Reply:
🤔 It will not, no.

I considered injecting the `Date` into that function to make more sophisticated unit tests but was not sure if it is worth it. With your comment it may be more worth it than I originally thought.